drivers/at86rf2xx/at86rf2xx_getset.c

```diff
@@ -172,12 +172,12 @@ void at86rf2xx_set_page(at86rf2xx_t *dev, uint8_t page)
 
 uint16_t at86rf2xx_get_pan(at86rf2xx_t *dev)
 {
-    return dev->pan;
+    return byteorder_ntohs(dev->pan);
 }
 
 void at86rf2xx_set_pan(at86rf2xx_t *dev, uint16_t pan)
 {
-    dev->pan = pan;
+    dev->pan = byteorder_htons(pan);
```

Comment:
Yes in the MAC header (which is the case here). I was talking about the handing of the option to the upper layers. But your comment solicited me to [read the doc](https://github.com/RIOT-OS/RIOT/blob/master/sys/include/net/netopt.h#L55) again and I was wrong: The PAN is supposed to be handed up in host byte order. I guess I was confused because the `ifconfig` config command of my implementation prints it address-like. Will fix this.